gunicorn/arbiter.py

```diff
@@ -535,7 +536,8 @@ def spawn_worker(self):
                 worker.tmp.close()
                 self.cfg.worker_exit(self, worker)
             except:
-                pass
+                self.log.error("Exception during worker exit:\n%s",
```

Comment:
warning should be indeed better since this error is harmless. @jgehrcke can you make the change?